hippod/mime_data_db.py

```diff
@@ -141,6 +141,32 @@ def decode_and_write_file(app, sha_sum, data, compress=False):
     return d
 
 
+def already_in_data_db(app, sha):
+    data_db = app['DB_DATA_PATH']
+    data_list = os.listdir(data_db)
+    if sha in data_list:
+        return True
+    return False
+
+
+def higher_mime_type_preference(app, sha, data):
```

Comment:
the last argument, data, should be a mimetype string, not the container object. This is more transparent, more transparent to the caller and for the callee.